opennms-doc/guide-install/src/asciidoc/text/opennms/introduction.adoc

```diff
@@ -3,92 +3,41 @@
+
+* _RHEL_: Red Hat Enterprise Linux 6 or higher, CentOS 6 or higher, Fedora 20 or higher
+* _Debian_: Debian 7 or higher, Ubuntu 14.04 or higher
+* _Microsoft Windows_: Windows 8.1, Windows Server 2012, Windows 10
+
+Installable, precompiled software packages are provided through _RHEL Yum_ and _Debian APT_ repository servers and from the
+link:https://sourceforge.net/projects/opennms/files/OpenNMS/[OpenNMS Sourceforge project page].
+Installing _OpenNMS_ requires the following prerequisites:
+
+* A configured <<gi-install-opennms-repo-releases, Yum or APT Package Repository>> for your platform (Linux only)
+* Internet access to download and verify _OpenNMS_ packages from the Yum or APT package repositories
+* <<gi-install-oracle-java, Oracle Java SE Development Kit 8>> environment
+* PostgreSQL database version 9.1 or higher
+* A time-series database engine to persist long-term performance data:
+** JRobin: The default choice. JRobin is included inside OpenNMS and doesn't require additional software to be installed.
+** <<gi-rrdtool-time-series-database, RRDtool>>: A higher performance, file-based database.
+** <<gi-install-ts-newts, Newts>>: The highest performance solution. Newts uses an Apache Cassandra database for clustered scalability.
+
+NOTE: _OpenJDK 8_ can be used, but for production and critical environments _Oracle Java SE Development Kit 8_ is recommended.
+
+[NOTE]
+====
+`${OPENNMS_HOME}` will be used to refer to the path where _OpenNMS_ is installed. It is different
+depending on your platform:
+
+* _RHEL_: `/opt/opennms`
+* _Debian_: `/usr/share/opennms`
+* _Microsoft Windows_: `C:\Program Files\opennms`
+====
 
 With the _opennms_ meta package all dependencies needed for the components mentioned above are maintained.
 The following sections describe how to install _OpenNMS_ on a single system.
-Dependencies for _Java_ and the _PostgreSQL_ data base are maintained with the _opennms_ meta installation package.
-
-[[gi-install-opennms-repo-releases]]
-=== Repositories for Releases
-
-Installation packages are available for different releases of _OpenNMS_.
-The configuration of the repository decides which _OpenNMS_ release will be installed.
-
-The following releases are available for installation:
-
-._OpenNMS_ release name convention
-[options="header, autowidth"]
-|===
-| Release                   | Description
-| `stable`                  | Latest stable release
-| `testing`                 | Release candidate for next stable
-| `snapshot`                | Latest successful develop build
-| `branches/${BRANCH-NAME}` | Install from a specific branch name, e.g. `branches/features-newts` installs the repository for the _Newts_ development branch.
-                              Branches can be found in http://yum.opennms.org/branches/ or http://debian.opennms.org/dists/branches/
-|===
-
-To install a different release the repository files have to be installed and manually modified.
-
-==== Specific Release on RHEL-based system
```

Comment:
Haven't found where do we describe how to setup the yum repository? Is this not necessary anymore?